src/test/java/io/vlingo/actors/WorldTest.java

```diff
@@ -7,18 +7,15 @@
 
 package io.vlingo.actors;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import org.junit.After;
 import org.junit.Test;
 
 import io.vlingo.actors.testkit.TestUntil;
 
+import static org.junit.Assert.*;
```

Comment:
BTW, do you use IntelliJ IDEA? I am currently using Eclipse and as soon as I do an "Organize Imports" it will expand all the package.* references  to explicit classes. That's why I generally don't package.* even though I prefer it. (Just a question, not a suggestion.)

Reply:
I use IDEA, I personally prefer it :D. But I can change my IDEA settings to avoid compacting packages if you think it's a good idea.

Reply:
@kmruiz No, it's ok. I think I was just apologizing ahead of time in case my Eclipse stubs your toe when I automatically reach for Shift+Command+O. That's correct, I am apologizing for Eclipse, not for me ;) Come to think of it, maybe Eclipse now has the option to compact on Organize Imports. Hmmm, will check.